service.upnext/service.py

```diff
@@ -0,0 +1,12 @@
+import xbmcaddon
+import xbmc
+import os
+from resources.lib.monitor import Monitor
+
+cwd = xbmcaddon.Addon(id='service.upnext').getAddonInfo('path').decode('utf-8')
+BASE_RESOURCE_PATH = xbmc.translatePath(os.path.join(cwd, 'resources', 'lib')).decode('utf-8')
```

Comment:
why do you need to append a path that belongs to your addon using the sys module?